tests/test_trace_reconstruction.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+from src.trace_reconstruction import *
+from src import trace_reconstruction
+import filecmp
+import shutil
+
+
+
```

Comment:
Too many blank lines

Reply:
resolved in [166cddb](https://github.com/INFB-Projekt/flashspeicher-rekonstruktion/pull/22/commits/166cddb3a6e82d5bd674912293c9f8ca219ac63e)